execution_engine/src/core/tracking_copy/tests.rs

```diff
@@ -282,11 +258,9 @@ fn tracking_copy_rw() {
     let value = StoredValue::CLValue(CLValue::from_t(3_i32).unwrap());
     let _ = tc.read(correlation_id, &k);
     tc.write(k, value.clone());
-    assert_eq!(tc.ops.len(), 1);
-    assert_eq!(tc.ops.get(&k), Some(&Op::Write));
```

Comment:
@Fraser999 why these two asserts got removed? if they are no longer true then I would prefer to update the rules instead.

Reply:
The `TrackingCopy` type no longer has the `ops` field.